### PR TITLE
Add & Fetch all Archive Tasks

### PR DESCRIPTION
**Pull request by pepps01**

### CODE REVIEW

File: app/Http/Controllers/TaskController.php [hunk 1]
```diff
@@ -135,4 +135,37 @@ public function sort(Request $request)
         return $collectionTasks;
     }
 
+    public function getArchivedTasks(Request $request)
```
Comment (EricOgie) [inline app/Http/Controllers/TaskController.php]: This method is getting all tasks and not archived tasks. Your issue is to Create an API to fetch all tasks that have been archived.  What you have done here won't serve the purpose.

File: app/Http/Controllers/TaskController.php [hunk 2]
```diff
@@ -135,4 +135,37 @@ public function sort(Request $request)
...
+    }
+
+
+    public function getArchivedTasksById(Request $request,$id)
+    {           
+        $tasks = ($this->taskService->find($id));
+         return response()->json($tasks);
+    }
+
+    public function archiveTask(Request $request, $id)
```
Comment (EricOgie) [inline app/Http/Controllers/TaskController.php]: I really don't understand why you put this method here. What you are to work on has no business with creating a task or what looks like it. Please understand the issue before writing your code.

**Comment by pepps01** [comment]
@eric… it’s fine.  I made a test. Please Ignore. I would make the changes.

File: routes/api.php [hunk 3]
```diff
@@ -53,10 +53,12 @@
 
 // api to fetch all todo tasks
 Route::get('task', [\App\Http\Controllers\TaskController::class, 'index']);
-
 // comment post request
 
 
+Route::get('archived-tasks', [\App\Http\Controllers\TaskController::class, 'getArchivedTasks']);
```
Comment (EricOgie) [inline routes/api.php]: A route is enough for the API you are to create. Having multiple routes for an API to just fetch archived tasks is, at best, redundant.

**Comment by EricOgie** [comment]
I have to close this. Please see my comments for reasons why I can't merge this.

**Comment by pepps01** [comment]
Can’t I make changes to this. Instead of closing it yet ?

**Pull request closed without merging**